src/types.ts

```diff
@@ -7,6 +7,8 @@ import { EASSchemaUid } from './identity/types';
  */
 export type OnchainKitConfig = {
   address: Address | null; // Address is optional as we may not have an address for new users
+  apiKey: string; // ApiKey is required to use CDP APIs
```

Comment:
People reading CDP, will have not clue what is it. Let's probablly spell it out.

Reply:
also, do we think we might need more explicit here? like `coinbaseApiKey`? this is more an open question. what do we think?

Reply:
I think `apiKey` makes sense. Don't people already associate OnchainKit with the Coinbase brand? No need to explicitly call out imo